[DOC-3254] Helm 2 deprecation notice partial file (#2092)
harness/developer-hub: release-notes/shared/helm-2-deprecation-notice.md

> To safeguard your operations and protect against potential security vulnerabilities, Harness will launch an update to deprecate the Helm 2 binary from delegates with an immutable image type (image tag yy.mm.xxxxx) on July 30, 2023. For delegate types, go to [Delegate image types](/docs/platform/delegates/delegate-concepts/delegate-image-types).

Reviewer: ```suggestion
To safeguard your operations and protect against potential security vulnerabilities, Harness will launch an update to deprecate the Helm 2 binary from delegates with an immutable image type (image tag yy.mm.xxxxx) on July 30, 2023. For information on delegate types, go to [Delegate image types](/docs/platform/delegates/delegate-concepts/delegate-image-types).
```

> Helm 2 was deprecated by the Helm Community in November 2020 and is no longer supported. Continuing to maintain the Helm 2 binary on your delegate introduces high and critical vulnerabilities, which may put your infrastructure at risk.
> 
> If your delegate is set to auto upgrade, Harness will automatically remove the binary from your delegate. This will result in pipeline and workflow failures for services deployed via Helm 2. 

Reviewer: ```suggestion
Helm 2 was deprecated by the Helm community in November 2020 and is no longer supported by Helm. If you continue to maintain the Helm 2 binary on your delegate, it might introduce high and critical vulnerabilities and put your infrastructure at risk.

If your delegate is set to auto-upgrade, Harness will automatically remove the binary from your delegate. This will result in pipeline and workflow failures for services deployed via Helm 2. 
```

> :::info note
> If your development team still uses Helm 2, you can re-introduce the binary on the delegate. Harness is not responsible for any vulnerabilities or risks that may result from re-introducing the Helm 2 binary.
> :::

Reviewer: ```suggestion
:::info note
If your development team still uses Helm 2, you can reintroduce the binary on the delegate. Harness is not responsible for any vulnerabilities or risks that might result from reintroducing the Helm 2 binary.
:::
```

> For more information about updating your delegates to re-introduce Helm 2, go to:

Reviewer: ```suggestion
For more information about updating your delegates to reintroduce Helm 2, go to:
```
